src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -896,5 +966,153 @@ public static void InvalidDateTimeStyles()
             DateTimeStyles style = DateTimeStyles.AssumeLocal | DateTimeStyles.AssumeUniversal;
             AssertExtensions.Throws<ArgumentException>("style", () => DateTime.ParseExact(strDateTime, formats, provider, style));
         }
+
+        public static IEnumerable<object[]> Parse_ValidInput_Suceeds_MemberData()
+        {
+            yield return new object[] { "1234 12", null, new DateTime(1234, 12, 1, 0, 0, 0) };
+            yield return new object[] { "12 1234", null, new DateTime(1234, 12, 1, 0, 0, 0) };
+            yield return new object[] { "12 1234 11", null, new DateTime(1234, 12, 11, 0, 0, 0) };
+            yield return new object[] { "1234 12 13", null, new DateTime(1234, 12, 13, 0, 0, 0) };
+            yield return new object[] { "12 13 1234", null, new DateTime(1234, 12, 13, 0, 0, 0) };
+            yield return new object[] { "1 1 1", null, new DateTime(2001, 1, 1, 0, 0, 0) };
+            yield return new object[] { "2 2 2Z", null, null };
```

Comment:
third parameter should be `new DateTime(2002, 2, 1, 16, 0, 0)` not `null` ?

Reply:
This one was on purpose. Wasn't sure of a good way to avoid hardcoding timezone-dependent state. If there's a good way, happy to change it.

Reply:
put "if PST" around the check maybe? then it will at least run sometimes